activestate.windows.yaml

```diff
@@ -24,7 +21,14 @@ scripts:
     if: eq .OS.Name "Windows"
     description: Copies install scripts to build dir, this script exists for transparency with CI behavior
     value: |
-      cp installers/install.ps1 build/install.ps1
+
+      BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+      INSTALLERS_DIR=build/installers/${BRANCH#release}
+      mkdir -p $INSTALLERS_DIR
```

Comment:
We'd still want this to go to the old location if it's the `release` branch.

Reply:
Yes, That's why i have this `${BRANCH#release}` part, so in the `release` case, it is copied to `./build/installers/install.sh|install.ps1` and from there to the same location where it used to go `s3://cli-update/update/state/install.sh|install.ps1`

Reply:
At least I hope that this will happen, we should confirm after the first deployment to `master` that the scripts are uploaded to `s3://cli-update/update/state/master/`.  After that is clear, I am confident, the release install script will end up in the correct place.